test/README.md

```diff
@@ -0,0 +1,294 @@
+# SQL-tester
+This is an introduction for the SQL-tester project.  
+
+## About
+SQL-tester is a SQL testing framework for StarRocks , designed to reduce the cost of test case construction. It uses SQL language to write test cases and has the ability to verify test results.
+
+### [**QuickStart**](#QuickStart)
+Documentation on getting started, installation and simple use.  
+
+### [**Run Parameters**](#Parameters)
+Detailed description of the execution parameters.  
+
+### [**Grammar**](#Grammar)  
+Reference documentation for the syntax format and other functions.
+
+
+# <span id="QuickStart">QuickStart</span>
+## Prerequisites
+### 1. Python
+
+<table>
+  <tr>
+    <td>Python</td>
+    <td>>= 3.6.8</td>
+  </tr>
+</table>
+
+### 2. Pip
+```shell
+yum -y install python-pip
+python3 -m pip install --upgrade pip
+```
+
+### 3. Requirements
+Information about the dependencies is stored in the `requirements.txt`.   
+You can install them by:  
+```shell
+$ python3 -m pip install -r requirements.txt
+```
+
+### 4. Config
+First you need to prepare a target sr to be tested, and then store their information in `conf/sr.conf`.  
```

Comment:
Is that possible to manage sr's fe/be process by using python scripts ? so can test more failover screnes.

Reply:
Yes, we already support this type of capability in the integration testing framework. Whether or not to inherit into the SQL framework will be considered in the future.